Resources/Prototypes/Recipes/Cooking/meal_recipes.yml

```diff
@@ -614,11 +616,59 @@
   solids:
     FoodOnionSlice: 1
 
+#Donuts
+- type: microwaveMealRecipe
+  id: RecipeDonut
+  name: donut recipe
+  result: RMCFoodDonut
+  time: 10
+  group: Breads
+  solids:
+    FoodDoughRope: 1 # created by rolling a dough slice.
+  reagents:
+    Sugar: 5
+
+- type: microwaveMealRecipe
+  id: RecipeDonutFrosted
+  name: frosted donut recipe
+  result: RMCFoodDonutFrosted
+  time: 10
+  group: Breads
+  solids:
+    FoodDoughRope: 1
+  reagents:
+    Sugar: 5
+    JuiceBerry: 5
+
+
+#Eggs
+- type: microwaveMealRecipe
+  id: RecipeOmelette
+  name: omelette recipe
+  result: FoodMealOmelette
+  time: 10
+  group: Eggs
+  solids:
+    FoodEgg: 1
+    FoodCheeseSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFriedEgg
+  name: fried egg recipe
+  result: FoodMealFriedegg
+  time: 10
+  group: Eggs
+  solids:
+    FoodEgg: 1
+  reagents:
+    TableSalt: 1
+    Blackpepper: 1
+
```

Comment:
New prototypes such as these should be in a new file under the _RMC14 folder, to separate it from the upstream files in order to minimize merge conflicts